beacon_node/src/cli.rs

```diff
@@ -816,6 +816,15 @@ pub fn cli_app() -> Command {
                 .action(ArgAction::Set)
                 .display_order(0)
         )
+        .arg(
+            Arg::new("state-cache-max-MiB")
+                .long("state-cache-max-MiB")
+                .value_name("MiB")
```

Comment:
Most users without CS degrees will recognise "megabytes" or MB

```suggestion
                .value_name("MEGABYTES")
```